pages/common/git-archive.md

```diff
@@ -1,28 +1,28 @@
 # git archive
 
-> Create an archive of files from a named tree.
+> Create an archive of files from a tree.
 > More information: <https://git-scm.com/docs/git-archive>.
 
 - Create a tar archive from the contents of the current HEAD and print it to `stdout`:
 
 `git archive --verbose HEAD`
 
-- Create a zip archive from the current HEAD and print it to `stdout`:
+- Use the `zip` format and report progress [v]erbosely:
 
-`git archive --verbose --format zip HEAD`
+`git archive -{{v|-verbose}} --format zip HEAD`
 
-- Same as above, but write the zip archive to file:
+- [o]utput the zip archive to a specific file:
 
-`git archive --verbose --output {{path/to/file.zip}} HEAD`
+`git archive -v -{{o|-output}} {{path/to/file.zip}} HEAD`
```

Comment:
I don't see an issue with using the `{{token_syntax}}` here with options. This is backwards compatible with existing syntax and behaviour, just a creative new use for it!

We have I do believe used the `a|b` syntax before too to indicate a choice of 1 of multiple options.

I  would just note that `-{o|-output}` is **not valid token syntax**, and **would probably break clients and confuse users.** Using the proper `{{token_syntax}}` is a must here.

Either this or the `[m]enomic` syntax in the description depending on the situation  seems fine to me.